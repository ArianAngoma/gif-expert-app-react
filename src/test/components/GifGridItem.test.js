import {shallow} from "enzyme";
import GifGridItem from "../../components/GifGridItem";

describe('Test to component <GifGridItem/>', () => {
    const title = 'one punch GIF by Adult Swim';
    const url = 'https://media2.giphy.com/media/VXJWhaO7afRe/giphy.gif?cid=9ca6d3b12blk4snunf6e74rkyaojo7rnjoz33ldsx2mcr7ev&rid=giphy.gif&ct=g';

    let wrapper = shallow(<GifGridItem title={title} url={url}/>);

    beforeEach(() => {
        wrapper = shallow(<GifGridItem title={title} url={url}/>);
    })

    test('Should display the component <GifGridItem/> correctly', () => {
        expect(wrapper).toMatchSnapshot();
    });
})