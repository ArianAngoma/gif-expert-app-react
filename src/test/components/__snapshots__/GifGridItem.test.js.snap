// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test to component <GifGridItem/> Should display the component <GifGridItem/> correctly 1`] = `
<div
  className="card animate__animated animate__fadeIn"
>
  <img
    alt="one punch GIF by Adult Swim"
    src="https://media2.giphy.com/media/VXJWhaO7afRe/giphy.gif?cid=9ca6d3b12blk4snunf6e74rkyaojo7rnjoz33ldsx2mcr7ev&rid=giphy.gif&ct=g"
  />
  <p>
    one punch GIF by Adult Swim
  </p>
</div>
`;
